proto/pdpb.proto

```diff
@@ -242,7 +242,8 @@ message RegionHeartbeatRequest {
     uint64 keys_read = 9;
     // Approximate region size.
     uint64 approximate_size = 10;
-    reserved 11;
+    // Approximate number of keys.
+    uint64 approximate_count = 11;
```

Comment:
11 is reserved for backward compatibility. You can use 13 instead.

Reply:
not 12?

Reply:
12 has already been used for field `interval`.